mlrun/projects/project.py

```diff
@@ -2048,7 +2219,7 @@ def _resolved_function(
             func,
         )
 
-    def _set_function(
+    def _set_function_helper(
```

Comment:
I think the previous name is fine
```suggestion
    def _set_function(
```